build_tools/testing/update_tflite_models.py

```diff
@@ -0,0 +1,40 @@
+from absl import app
+from absl import flags
+from google.cloud import storage
+from google_auth_oauthlib import flow
+
+import tempfile
+import urllib
+
+FLAGS = flags.FLAGS
+
+flatbuffers = dict({
+    "mobilenet_v1.tflite":
+        "https://tfhub.dev/tensorflow/lite-model/mobilenet_v1_1.0_160/1/default/1?lite-format=tflite",
+    "posenet_i8.tflite":
+        "https://tfhub.dev/google/lite-model/movenet/singlepose/lightning/tflite/int8/4?lite-format=tflite",
+})
+
+BUCKET_NAME = "iree-model-artifacts"
+FOLDER_NAME = "tflite-integration-tests"
+
+
+def upload_model(source, destination, tmpfile):
+  """Uploads a file to the bucket."""
+  urllib.request.urlretrieve(source, tmpfile)
+
+  storage_client = storage.Client()
+  bucket = storage_client.get_bucket(BUCKET_NAME)
+  blob = bucket.blob("/".join([FOLDER_NAME, destination]))
+  blob.upload_from_filename(tmpfile)
+
+
+def main(argv):
+  tf = tempfile.NamedTemporaryFile()
+
+  for dst, src in flatbuffers.items():
+    upload_model(src, dst, tf.name)
```

Comment:
Please add a comment to this file explaining what the script is for and giving sample usage instructions. Does this require special permissions to run? Should it be run from the `main` branch or is it okay to run from personal branches that have diverged? etc.

Reply:
Added some details for uploading. Like anything with merge conflicts there could be issues, not sure what to detail outside of that. I am hoping that the file can more be "documentation of source" for each file. Fortunately the file bails out if a original file does not exist so to my knowledge we should never wipe out the uploads if someone reruns.